dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -144,15 +145,17 @@ private static void setMetadataKeywords(DepositMetadata depositMetadata, Workflo
      * @param depositMetadata Metadata for the workflow version
      * @param workflow    workflow for which DOI is registered
      */
-    private static void setMetadataRelatedIdentifiers(DepositMetadata depositMetadata, Workflow workflow) {
+    private static void setMetadataRelatedIdentifiers(DepositMetadata depositMetadata, String dockstoreUrl,
+            Workflow workflow) {
         // Get the aliases for this workflow and add them to the deposit
         // The alias must be a format supported by Zenodo such as
         // DOI, Handle, ARK...URNs and URLs
         // See http://developers.zenodo.org/#representation 'related identifiers'
         List<RelatedIdentifier> aliasList = workflow.getAliases().keySet().stream()
-                .map(s -> {
+                .map(alias -> {
                     RelatedIdentifier relatedIdentifier = new RelatedIdentifier();
-                    relatedIdentifier.setIdentifier(s);
+                    String aliasUrl = dockstoreUrl + "/aliases/workflows/" + alias;
+                    relatedIdentifier.setIdentifier(aliasUrl.toString());
```

Comment:
aliasUrl is already a String; no need to call `toString`

Reply:
done